server/common/log-shared/src/main/java/org/kaaproject/kaa/server/common/log/shared/avro/gen/RecordHeader.java

```diff
@@ -19,6 +19,7 @@
  *
  * DO NOT EDIT DIRECTLY
  */
+
```

Comment:
I  separated license from class, this does not affect code, just for pretty look.